src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseReplicationTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ScenarioTest.SqlTests;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
+{
+    public class DatabaseReplicationTests : SqlTestsBase
+    {
+        [Fact]
+        [Trait(Category.Sql, Category.CheckIn)]
```

Comment:
None of these will be run as part of check-in.  Please change the trait to

```c#
[Trait(Category.AcceptanceTyoe, Category.CheckIn)]
```

for those tests that should be run as check-in tests.